### PR TITLE
LTD:-3357: Update routing sha

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description

- Removes routing engine functions that are now moved to `lite-routing`

### CODE REVIEW

File: api/workflow/tests/test_automated_case_routing.py [hunk 1]
```diff
@@ -120,292 +117,3 @@ def test_good_query_returns_parameter_set(self):
         parameter_set = case.parameter_set()
 
         self.assertIn(case.case_type, parameter_set)
-
-
-class CaseRoutingAutomationTests(DataTestClient):
```
Comment (saruniitr) [inline api/workflow/tests/test_automated_case_routing.py:125]: these are in `lite-routing` now

File: api/workflow/tests/test_user_queue_assignment.py [hunk 2]
```diff
@@ -123,65 +123,3 @@ def test_countersigning_queue_multiple_feeder_queues(self):
         self.assertIn(countersigning_queue, self.case.queues.all())
         self.assertNotIn(feeder_queue_2, self.case.queues.all())
         self.assertEqual(self.case.status, old_status)
-
-
-class NextStatusTests(DataTestClient):
```
Comment (saruniitr) [inline api/workflow/tests/test_user_queue_assignment.py:128]: moved to lite-routing

File: api/workflow/user_queue_assignment.py [hunk 3]
```diff
@@ -41,29 +42,7 @@ def get_next_compliance_visit_status(case):
         return CaseStatus.objects.get(status=CaseStatusEnum.compliance_visit_statuses[current_status_pos + 1])
 
 
-def get_next_status_in_workflow_sequence(case):
```
Comment (saruniitr) [inline api/workflow/user_queue_assignment.py:44]: moved to lite-routing

**Review by kevincarrogan — APPROVED** [review]

File: api/workflow/tests/test_user_queue_assignment.py [hunk 4]
```diff
@@ -123,65 +123,3 @@ def test_countersigning_queue_multiple_feeder_queues(self):
...
-        ]
-    )
-    def test_next_status_increments(self, old_status, new_status):
-        self.case.status = CaseStatus.objects.get(status=old_status)
-        self.case.save()
-        result = get_next_status_in_workflow_sequence(self.case)
-        self.assertEqual(result.status, new_status)
-
-
-class NextStatusGoodsQueryTests(DataTestClient):
```
Comment (hnryjmes) [inline api/workflow/tests/test_user_queue_assignment.py:159]: should this be deleted?

**Review by currycoder — APPROVED** [review]
Nice one!

**Review by hnryjmes — CHANGES_REQUESTED** [review]
I don't see where the class `NextStatusGoodsQueryTests` and its tests are in lite-routing so want to check if they've been deleted by mistake?

File: api/workflow/tests/test_user_queue_assignment.py (hunk 4 above)
Comment (saruniitr) [reply api/workflow/tests/test_user_queue_assignment.py:159]: Yes, when I moved `get_next_status_in_workflow_sequence()` to `lite-routing` I have include one more change as part of it.
I have removed support for certain case types which we don't support yet.

I could've moved as is but used this opportunity to remove some of the unused code hence these tests are no longer required.

**Review by hnryjmes — APPROVED** [review]

**Pull request merged**